prisma/schema.prisma

```diff
@@ -0,0 +1,34 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+// Looking for ways to speed up your queries, or scale easily with your serverless or edge functions?
+// Try Prisma Accelerate: https://pris.ly/cli/accelerate-init
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "mysql"
+  url      = env("DATABASE_URL")
+}
+
+model user {
+  id           Int       @id @default(autoincrement())
+  username     String
+  email        String
+  passkey_type String
```

Comment:
Since name on model would affect the code we wrote, it could be better to follow TS convention. Maybe we could follow their convention.

Model: https://www.prisma.io/docs/orm/reference/prisma-schema-reference#model
Model fields: https://www.prisma.io/docs/orm/reference/prisma-schema-reference#model-fields